pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ASTModifierNode.java

```diff
@@ -23,125 +28,133 @@ public Object jjtAccept(ApexParserVisitor visitor, Object data) {
         return visitor.visit(this, data);
     }
 
+    private static final Map<Keyword, Integer> opcodes = new HashMap<>();
+
+    static {
```

Comment:
remind me -- we can't depend on Guava, right?

Reply:
PMD does have an existing Guava dependency. How do you recommend that I use it here?

Reply:
Guava has proper support for ImmutableMap, which is a good choice in all sorts of situations, and would make this a more constant-y constant. It also has a syntax for a one-line declaration/assignment which would eliminate the `static` block here.